src/pages/en/guides/cms/storyblok.md

```diff
@@ -94,29 +94,29 @@ export default defineConfig({
 
 The Storyblok integration requires an object with the following properties:
 
-1. `accessToken` - This references the Storyblok API tokens that we added in the previous step.
+1. `accessToken` - This references the Storyblok API token that you added in the previous step.
 
     :::tip
     Since the Astro config file does not normally support environment variables, use the `loadEnv` function from Vite to load them.
     :::
 
-2. `components` - An object that maps Storyblok component names to paths to your local components. This is required to render your Storyblok components in Astro.
+2. `components` - An object that maps Storyblok component names to paths to your local components. This is required to render your Storyblok Bloks in Astro.
 
     :::note
-    The component paths are relative to the `src` directory.
+    The component paths are relative to the `src` directory. For example, if your component is located at `src/storyblok/MyComponent.astro`, the path would be `storyblok/MyComponent` (without the `.astro` extension).
     :::
 
 3. `apiOptions` - An object containing [Storyblok API options](https://github.com/storyblok/storyblok-astro#options). 
 
     :::caution
-    The default region is `eu`. If your Storyblok space was created in the US region, you will need to set the `region: 'us'`.
+    By default, the region is `eu`. If your Storyblok space was created in the US region, you will need to set the region to `us`.
```

Comment:
```suggestion
    By default, the region is `eu`. Other available regions are ...
```
I'll let you suggest a wording based on what's actually true, but I know other services let you choose an EU or US instance, but it's not actually *required* based on where you live. E.g., I think some services let me choose which servers I want my data stored in, and I might choose that based on privacy laws.

So, is this a thing where you HAVE to set `us` or your service doesn't work? And, what if I'm in Canada?  😅 Am I supposed to choose `us` because it's closer to me? Is `us` the only other option besides `eu`? If there are other options available, then it makes more sense to list what they are and say that you can choose a different one if... you want to?

Reply:
I see your point thanks for bringing this up. In this case, this setting depends on where the user created its content space, not where the user is located. There is only two options "us" and "eu".

I decided to include this since the first time I used the integration, I had no idea that this option existed. So I created my content space in the US and set up my project, and it didn't work. I was stuck on this one for some time, the error only suggested that the fetch request failed. So I tried everything: getting a new key, rewriting the fetch request, and checking the component mapping. I don't remember how I found this option but I decided to try and it worked.

Hence, my thinking was if I got stuck on this one, there is a high chance someone else might get stuck on this one too, so lets mention it.

Reply:
Yes, absolutely let's metion it then! So, I wonder what happens if you're not in either of those locations? I wonder how it chooses which one you have to use?  Is it ONLY if you're in the US? Like, what do you think happens if you're in Canada?

Let's keep it like this for now, because it's a good warning, but we might want to revisit this. I'm not sure at this exact moment how to delicately say "STORYBLOK KNOWS WHERE YOU LIVE" lol.

Reply:
Sorry, I forgot to mention that whenever you create a new content space, you can select a region from the dropdown. 
![image](https://user-images.githubusercontent.com/46791833/205349654-8280f345-a1af-4baf-9318-fb8b6b251857.png)

Reply:
I think the instruction is good as is then! I'm happy with this whenever you want to merge! It is FRIDAY afterall!  🥳